lib/salesforce_sync/resource/action.rb

```diff
@@ -1,16 +1,39 @@
 module SalesforceSync
   module Resource
     class Action
+      SF_USER_QUERY_TYPE = "Contact"
```

Comment:
Not very happy with this close knowing details of the user type but not sure what else we can do